doc/source/templates/04_finetuning_llms_with_deepspeed/README.md

```diff
@@ -2,7 +2,7 @@
 | Template Specification | Description |
 | ---------------------- | ----------- |
 | Summary | This template, demonstrates how to perform fine-tuning (full parameter or LoRA) for Llama-2 series models (7B, 13B, and 70B) using TorchTrainer with the DeepSpeed ZeRO-3 strategy. |
-| Time to Run | ~14 min. for 7B for 1 epoch on 3.5M tokens. ~26 min for 13B for 1 epoch.  |
+| Time to Run | ~14 min for 7B for 1 epoch on 3.5M tokens; ~26 min for 13B for 1 epoch on 3.5M tokens; ~190 min for 70B for 1 epoch on 3.5M tokens (see the setup details below)  |
```

Comment:
1 epoch (3.5M tokens) training wall-clock time: ~14 min. for 7B, ~26 min. for 13B, and ~190 min. for 70B

Reply:
Much cleaner. Updated. Thx!